x-pack/legacy/plugins/canvas/common/lib/hex_to_rgb.test.ts

```diff
@@ -4,21 +4,21 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { hexToRgb } from '../hex_to_rgb';
+import { hexToRgb } from './hex_to_rgb';
 
 describe('hexToRgb', () => {
-  test('invalid hex', () => {
+  it('invalid hex', () => {
```

Comment:
minor but if we're switching to use the `it` syntax, I think we should make the descriptions match that. For example, changing `invalid hex` here to `should return null for invalid hex values`